pkg/operator/collection.go

```diff
@@ -43,6 +43,10 @@ import (
 	monitoringv1alpha1 "github.com/GoogleCloudPlatform/prometheus-engine/pkg/operator/apis/monitoring/v1alpha1"
 )
 
+var (
+	TrueVar = true
```

Comment:
Shouldn't be exported. If you put it at the package level instead of function level, I'd also make it a function to ensure it cannot be modified.

Reply:
Done.